webapp/views.py

```diff
@@ -144,18 +146,15 @@ def create_asset():
     """
     Create a new asset
     """
-    asset = request.files.get("asset")
-    file_content = asset.read()
+    asset = request.values.get("asset")
```

Comment:
What changed here? This used to be submitted as a file through a standard form and now it's submitted as a base64 string? Why is that?

Reply:
the standard form case is now part of the manager create view as for the base64 according to [this code](https://github.com/canonical/canonicalwebteam.upload-assets/blob/main/canonicalwebteam/upload_assets/uploader.py#L33-L43) from the upload-assets this is sent as base64.

Reply:
But I don't think I have the full story here. You say this is to fix upload-assets. Which used to work. So what changed? Is this a change to the old API spec? Or is it to set it back to what the old API spec used to be?

Reply:
the second one, set it back to the old API spec: https://github.com/canonical/assets.ubuntu.com/blob/legacy/webapp/views.py#L164